src/components/pages/AddFarmer/AddFarmer.jsx

```diff
@@ -0,0 +1,251 @@
+/** @format */
+
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+export class AddFarmer extends Component {
+  state = {
+    farmer: {},
+  }
+  render() {
+    return (
+      <div>
+        <header>
+          <div>Logo</div>
+          <div>
+            <button>Logout</button>
+          </div>
+        </header>
+
+        <section>
+          <button>Back</button>
+          <hr />
+
+          <form action="">
+            <div>
+              <h1>Personnel Information</h1>
+              <div>
+                <label for="">
+                  Title:
+                  <select id="" name="title">
+                    <option value="">Miss</option>
+                    <option value="">Mrs</option>
+                    <option value="">Mr</option>
+                    <option value="">Chief</option>
```

Comment:
Is the chief title a thing in ghana? I could only find info about it for Nigeria

Reply:
that's what's in the database scheme... kindly confirm from shola